runtime/bindings/python/iree/runtime/_binding.py

```diff
@@ -12,8 +12,41 @@
 the way this trampoline functions.
 """
 
+import os
 import sys
+import warnings
 
-from iree import _runtime
+# Detect which runtime variant we are using.
+variant = os.getenv("IREE_PY_RUNTIME", "default")
+if variant == "default":
+    import iree._runtime_libs as _libs
 
+    try:
+        import iree._runtime_libs.version as _version
+    except ModuleNotFoundError:
+        _version = None
+    from iree._runtime_libs import _runtime
+elif variant == "tracy":
+    try:
+        import iree._runtime_libs_tracy as _libs
+        from iree._runtime_libs_tracy import _runtime
+    except ModuleNotFoundError as e:
+        raise ModuleNotFoundError(
+            "IREE Tracy runtime requested via IREE_PY_RUNTIME but it is not "
+            "enabled in this build"
+        ) from e
+    try:
+        import iree._runtime_libs.version as _version
+    except ModuleNotFoundError:
+        _version = None
+    print("-- Using Tracy runtime (IREE_PY_RUNTIME=tracy)", file=sys.stderr)
+else:
+    warnings.warn(
+        f"Unknown value for IREE_PY_RUNTIME env var ({variant}): " f"Using default"
+    )
```

Comment:
This branch skips over the imports in the "default" branch... is this message accurate?

One idea for reordering:

```python
variant = os.getenv("IREE_PY_RUNTIME", "default")
if variant == "tracy":
  # ... tracy handling
else:
  if variant != "default":
    warnings.warn(
        f"Unknown value for IREE_PY_RUNTIME env var ({variant}): " f"Using default"
    )
  # ... default handling
```

Reply:
good catch. thanks.